server/app.py

```diff
@@ -1,11 +1,59 @@
-from flask import Flask
+import json
+from flask import Flask, jsonify, request
+from flask_sqlalchemy import SQLAlchemy
 from api.ping_handler import ping_handler
 from api.home_handler import home_handler
 
 
 app = Flask(__name__)
+# username & password: postgres
+# db name: mentionscrawler
+app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql://postgres:postgres@localhost/mentionscrawler'
```

Comment:
I thought we weren't going to have authentication for the database right now? Also do we need to specify that we're using pscopg2 or does the flask version of sqlalchemy specify that for us? (I only messed around with the standard sqlalchemy package) shouldn't it be 'postgresql+psycopg2:///mentionscrawler'?

Reply:
It worked for me without specifying psycopg2 so I guess sqlalchemy uses it by default. I forgot about the authentication. Just to clarify it should be
'postgresql://postgres@localhost/mentionscrawler' right?

Reply:
I think it might default to it, I just like it for clarity, and no it should be 'postgresql:///mentionscrawler' otherwise it will yell at you for not providing a password (at least it did for me when I tried to do it) plus that assumes you're logging in with the default postgres username